s3auth-hosts/src/main/java/com/s3auth/hosts/MkAmazonS3.java

```diff
@@ -126,6 +126,7 @@
  * @checkstyle BracketsStructure (1000 lines)
  * @checkstyle LineLength (1000 lines)
  * @checkstyle ClassFanOutComplexity (1000 lines)
+ * @checkstyle MethodCountCheck (103 methods)
```

Comment:
@krzyk And how can I prevent Qulice from complaining about more than 100 methods?